extensions/netty-loom-adaptor/deployment/src/main/java/io/quarkus/netty/loom/adaptor/NettyLoomAdaptorProcessor.java

```diff
@@ -577,16 +595,310 @@ public MethodVisitor visitMethod(
                 final String signature,
                 final String[] exceptions) {
             if (cv != null) {
+                MethodVisitor mv = cv.visitMethod(access, name, descriptor, signature, exceptions);
```

Comment:
Due to the complexity and length of the generated bytecode, I would write in a comment the Java (or pseudo-Java) code corresponding to the bytecode.

Reply:
@geoand teach me about `-DskipTests -Dquarkus.package.quiltflower.enabled=true` and can be used to copy/paste the resulting Java code indeed

Reply:
Yeah, having the generated code in comments has saved me multiple times :)

Reply:
Another slightly unrelated trick when writing ASM: 

Write what you want in Java using IntelliJ, build the project and use ASM Bytecode Viewer plugin